src/libraries/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -4123,7 +4110,7 @@ public static void DCS_MyPersonSurrogate()
     }
 
     [Fact]
-    [ActiveIssue("https://github.com/dotnet/runtime/issues/85690", TestPlatforms.Wasi)]
+    //[ActiveIssue("https://github.com/dotnet/runtime/issues/85690", TestPlatforms.Wasi)]
     public static void DCS_FileStreamSurrogate()
```

Comment:
Should this be removed instead of commented out?

Reply:
Yup. My first pass just commented them. I thought I went back and cleaned them all up, but apparently missed this file. :/ Fixed now.